data/helpers.d/utils

```diff
@@ -24,3 +24,19 @@ ynh_get_plain_key() {
         fi
     done
 }
+
+# Curl abstraction to simplify its call
+#
+# Helpfully mostly for curl arguments
```

Comment:
In my mind, developer refer to the upstream dev of the app.
Packager refer only to the dev who work on the scripts for YunoHost.

"fiddle", I don't know. But why not I think it's also clear.